api/client.php

```diff
@@ -1,4 +1,5 @@
 <?php
+    require_once '../config.php';
```

Comment:
I would like to argue more for this code style. This kind of relative path import is not good for further code maintenance. We can change the Server configuration. We can also set up some kind of contant like `CAPU_ROOT` to the root directory for us.